src/main/resources/openapi/viewer-schemas.yaml

```diff
@@ -231,46 +231,47 @@ components:
           type: integer
           format: int64
           readOnly: true
-        name:
-          type: string
-        alias:
-          type: string
-        longname:
-          description: 'has featuretype name and name combined with a dot.'
-          example: 'v_factsheet_terrein_info.b_minhuur'
-          type: string
-        editAlias:
+        key:
```

Comment:
Why rename to `key` instead of `name`?

Reply:
ColumnMetadata also uses key, this makes the api more consistent.

https://github.com/B3Partners/tailormap-api/blob/8a4c185b4fe1595e83c80dc474fee5b3f5b3520c/src/main/resources/openapi/viewer-schemas.yaml#L314